mne_connectivity/base.py

```diff
@@ -419,10 +415,9 @@ def __init__(
     ):
         if isinstance(indices, str) and indices not in ["all", "symmetric"]:
             raise ValueError(
-                f"Indices can only be "
-                f'"all", otherwise '
-                f"should be a list of tuples. "
-                f"It cannot be {indices}."
+                "Indices can only be "
+                '"all", otherwise '
+                f"should be a list of tuples. It cannot be {indices}."
```

Comment:
```suggestion
                'Indices can only be "all", otherwise '
                f"should be a list of tuples. It cannot be {indices}."
```

also wondering why `"symmetric"` isn't mentioned in the Error as a valid option?

Reply:
@drammock How's this?

https://github.com/mne-tools/mne-connectivity/blob/69c1116506eeb48d2c0aa7cf5d9473cbd227429f/mne_connectivity/base.py#L416-L420